src/handlers/dashboard/get_chart_data.py

```diff
@@ -29,7 +29,8 @@ def _get_table_cols(table_name: str, version: str = None) -> list:
     s3_key = f"{prefix}/{version}/{table_name}__aggregate.csv"
     s3_client = boto3.client("s3")
     s3_iter = s3_client.get_object(
-        Bucket=s3_bucket_name, Key=s3_key  # type: ignore[arg-type]
+        Bucket=s3_bucket_name,
+        Key=s3_key,  # type: ignore[arg-type]
```

Comment:
Curious - what is the type: ignore doing there - is that a standardized hint convention to IDEs?

Reply:
its been a Hot Minute, but my recollection is that the AWS stuff wasn't type hinted correctly at the time.

Reply:
looks ok now - removed.